crates/example-types/src/node_types.rs

```diff
@@ -70,24 +70,24 @@ pub type StaticMembership = StaticCommittee<TestTypes>;
 
 impl<TYPES: NodeType> NodeImplementation<TYPES> for PushCdnImpl {
     type QuorumNetwork = PushCdnNetwork<TYPES>;
-    type CommitteeNetwork = PushCdnNetwork<TYPES>;
+    type DANetwork = PushCdnNetwork<TYPES>;
```

Comment:
I think we may want to call this `DaNetwork` based on https://rust-lang.github.io/api-guidelines/naming.html (`Uuid` vs `UUID`).

(I personally dislike this convention also, but I think we agreed on it in Zulip)